### PR TITLE
Updated heading levels, + misc

### PR DESCRIPTION
**Pull request by brianburrier**
1. Removed some legacy code
2. Updated heading levels from H3s to H2s. Will need to update the SDS Master Site after the merge
3. Needed to update heading levels in the script for now until the backend is updated
4. Wrapper footer text in container to target that instead

### CODE REVIEW

**Comment by brianburrier** [comment]
Pages to review:
https://stelterstaging.ingeniuxondemand.com/brian-sds-dev
https://stelterstaging.ingeniuxondemand.com/brian-sds-dev/wills-and-living-trusts
https://stelterstaging.ingeniuxondemand.com/brian-sds-dev/charitable-gift-annuities

File: StandardLayout/css/stelter-standard-a0000000-r0.css [hunk 1]
```diff
@@ -199,25 +197,23 @@
...
 	/* General Content */
-	.sub-middle-section p, .sub-middle-section table, .sub-middle-section tr, .sub-middle-section td, .sub-middle-section form, .sub-middle-section label, .sub-middle-section blockquote {font:inherit; color:inherit;}
+	.sub-middle-section p, .sub-middle-section table, .sub-middle-section tr, .sub-middle-section td, .sub-middle-section form, .sub-middle-section label, .sub-middle-section blockquote {}
 	.sub-middle-section p, .sub-middle-section table, .sub-middle-section form {margin:0 0 1.5rem;}
 	.sub-middle-section blockquote {margin:0 20px 1.5rem; padding:3px 15px; border-left:4px solid #ddd;}
     .sub-middle-section p.intro-copy {}
 
 	/* Match the color, margins/padding, font size, etc... to the client's site for all headers */
-	h1, .h1, h2, .h2, h3, .h3, h4, .h4 {} /* Headings Reset */
+	h1, .h1, h2, .h2, h3, .h3, h4, .h4 {margin-top:0;} /* Headings Reset */
```
Comment (chrisbutzke) [inline StandardLayout/css/stelter-standard-a0000000-r0.css]: Good idea!

File: StandardLayout/css/stelter-standard-a0000000-r0.css [hunk 2]
```diff
@@ -199,25 +197,23 @@
...
 	.stl-page-header {font-family:inherit; font-size:3.2rem; color:inherit;} /* Page header (not always used) */
 	.stl-page-title {font-family:inherit; font-size:3.2rem; color:inherit;} /* Page title styles */    
-        .subpage .stl-page-title {margin-bottom:5px;}
-	.stl-page-subtitle {font-family:inherit; font-size:2.4rem; color:inherit;} /* Sub Title styles */
+	.stl-page-subtitle {font-family:inherit; font-size:2.4rem; color:inherit; font-weight:700;} /* Sub Title styles */
         .subpage #stl-cms .stl-page-subtitle {margin-bottom:30px;}
-	.sub-middle-section h3, .main-copy h3, .main-copy h2 {font-family:inherit; font-size:2.0rem; color:inherit; margin-top:0;}
+	.sub-middle-section h3, .main-copy h3 {}
```
Comment (chrisbutzke) [inline StandardLayout/css/stelter-standard-a0000000-r0.css]: Let's add:
.sub-middle-section h2, .main-copy h2 {}

File: StandardLayout/css/stelter-standard-a0000000-r0.css [hunk 3]
```diff
@@ -199,25 +197,23 @@
...
 
 	/* If interior pages use the two column layout */
 	.interior-one-col .stl-page-title {text-align:left; max-width:990px; margin:0 auto 10px; padding:0 15px;}
 	.interior-one-col .stl-page-subtitle {text-align:left; max-width:990px; margin:0 auto 15px; padding:0 15px;}
-	.interior-two-col .stl-page-title {width:100%; flex:0 0 auto;}
-	.interior-two-col .stl-page-subtitle {width:100%; flex:0 0 auto; margin:0 0 15px;}
+	.interior-two-col header {width:100%; flex:0 0 auto;}
```
Comment (chrisbutzke) [inline StandardLayout/css/stelter-standard-a0000000-r0.css]: add:
.interior-one-col header {...}

File: StandardLayout/css/stelter-standard-a0000000-r0.css [hunk 4]
```diff
@@ -286,21 +282,21 @@
 	.stl-header .container {display:flex; flex-direction:row; justify-content:space-between; align-items:center; padding:20px 15px;}
 	.client-logo {display:block;}
 
-	.stl-footer {padding:20px 15px 10px;}
 	.stl-footer p {font-size:1.4rem;}
 	.stl-footer p a {color:inherit;}
     .stl-footer #stl-question-widget-form p {font-size:inherit;}
 
-	#stl-copyright-container, #stl-tax-id-container {max-width:1200px; margin:0 auto;}
+	.stl-footer-text {max-width:1200px; margin:0 auto; padding:3.0rem .15rem; max-width:1200px;}
```
Comment (chrisbutzke) [inline StandardLayout/css/stelter-standard-a0000000-r0.css]: If we wrap each of the footer items (other than the container component) in .container class, we shouldn't need a special class.

File: StandardLayout/css/stelter-standard-a0000000-r0.css [hunk 5]
```diff
@@ -409,11 +405,9 @@
 	/* update with client specific background-image */
 	#stl-banner {display:block; min-height:180px; background-image:url(/documents/Stelter/sds/images/home-banner.jpg); background-position:center center; background-repeat: no-repeat; background-size:cover; display:flex; flex-direction:column;}
 
-    @media screen {
-        .stl-top {position:relative;}
-        .stl-intro {display:block; margin:40px auto; padding:40px 15px; text-align:center; background:rgba(255,255,255,.7);}
-        .stl-intro p {max-width:750px; margin:0 auto 20px;}
-    }
+    .stl-top {position:relative;}
```
Comment (chrisbutzke) [inline StandardLayout/css/stelter-standard-a0000000-r0.css]: Keep inside @media screen {} so print style isn't affected.

File: StandardLayout/css/stelter-standard-a0000000-r0.css (hunk 2 above)
Comment (brianburrier) [reply StandardLayout/css/stelter-standard-a0000000-r0.css]: For now we'll keep .sub-middle-section h2, but I'll make a note that after the SDS MS is updated, we'll only need .main-copy h2.

File: StandardLayout/css/stelter-standard-a0000000-r0.css (hunk 3 above)
Comment (brianburrier) [reply StandardLayout/css/stelter-standard-a0000000-r0.css]: Looking more into the inspector and going back and forth between one and two column layouts, looks like this is what that section needs to be updated to:

```suggestion
	/* If interior pages use the one column layout */
	.interior-one-col header {max-width:990px; margin:0 auto 10px; padding:0 15px;}

	/* If interior pages use the two column layout */
	.interior-two-col header {width:100%; flex:0 0 auto;}
```

File: StandardLayout/css/stelter-standard-a0000000-r0.css (hunk 4 above)
Comment (brianburrier) [reply StandardLayout/css/stelter-standard-a0000000-r0.css]: Good point!

File: StandardLayout/css/stelter-standard-a0000000-r0.css (hunk 5 above)
Comment (brianburrier) [reply StandardLayout/css/stelter-standard-a0000000-r0.css]: Good catch.

**Pull request merged**